Investigate increased errors in run 36

This marathon run finally completed. There are a lot more errors:

```
wtwhite@wtwhite-vuw-vm:~/code/jcompile$ find jars -name '*.done'|wc -l
1792
wtwhite@wtwhite-vuw-vm:~/code/jcompile$ find jars -name '*.jar'|wc -l
2120
wtwhite@wtwhite-vuw-vm:~/code/jcompile$ find jars -name '*.error'|wc -l
601
```

Compared to last time:
```
wtwhite@wtwhite-vuw-vm:~/code/jcompile$ find runs/34_with_eq_from_dot_and_new_jep181/jars/EQ -name '*.done'|wc -l
1792
wtwhite@wtwhite-vuw-vm:~/code/jcompile$ find runs/34_with_eq_from_dot_and_new_jep181/jars/EQ -name '*.jar'|wc -l
2492
wtwhite@wtwhite-vuw-vm:~/code/jcompile$ find runs/34_with_eq_from_dot_and_new_jep181/jars/EQ -name '*.error'|wc -l
406
```

_Originally posted by @wtwhite in https://github.com/binaryeq/jcompile/issues/93#issuecomment-2205470512_

Just noticed that not rebuilding earlier `ecj` versions is problematic: I still added `-Dmaven.compiler.source=$JDK_MAJOR_VERSION` to them, which may affect things. In the case of `ecj-3.15.1`, it does: We see the same failure that we fixed elsewhere:

```
wtwhite@wtwhite-vuw-vm:~/code/jcompile/runs/37_extend_36_with_matching_docker_images_for_ecj$ grep "Failed to run the ecj compiler: Compliance level '.*' is incompatible with target level '.*'. A compliance level '.*' or better is required" jars/EQ/*/*.error
jars/EQ/ecj-3.15.1_openjdk-11.0.19/bcel-6.4.0.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project bcel: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/bcel-6.4.1.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project bcel: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/bcel-6.5.0.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project bcel: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/bcel-6.6.0.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project bcel: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/bcel-6.6.1.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project bcel: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/bcel-6.7.0.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project bcel: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-csv-1.10.0.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-csv: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-csv-1.6.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-csv: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.7' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-csv-1.7.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-csv: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-csv-1.8.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-csv: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-csv-1.9.0.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-csv: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-io-2.10.0.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-io: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-io-2.11.0.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-io: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-io-2.12.0.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-io: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-io-2.9.0.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-io: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-lang3-3.10.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-lang3: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-lang3-3.11.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-lang3: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-lang3-3.12.0.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-lang3: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-lang3-3.8.1.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-lang3: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.7' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-lang3-3.8.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-lang3: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.7' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
jars/EQ/ecj-3.15.1_openjdk-11.0.19/commons-lang3-3.9.jar.error:[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.0:compile (default-compile) on project commons-lang3: Fatal error compiling: Failed to run the ecj compiler: Compliance level '1.8' is incompatible with target level '9'. A compliance level '9' or better is required -> [Help 1]
```

**Workaround:** Rerun `ecj-3.15.1` builds, and get rid of `-Dmaven.compiler.source=$JDK_MAJOR_VERSION` from `java-compilers.json` for the two earlier `ecj` versions as its absence does not cause build failures for them. This means `java-compilers.json` will be "uneven", but it will at least reflect the jars we have built.

After discussion with @jensdietrich, we'll stick with the "uneven" `java-compilers.json` for now, but make an issue and mention it in the README.

---

Rerunning all `ecj-3.15.1` builds this morning didn't actually change anything, since the same builds that failed before still fail, just for a different reason, which doesn't look easy to solve. Example:

```
wtwhite@wtwhite-vuw-vm:~/code/jcompile/runs/37_extend_36_with_matching_docker_images_for_ecj$ cat jars/EQ/ecj-3.15.1_openjdk-11.0.19/bcel-6.4.0.jar.error
--snip--
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] [ecj] The compiler reported an error but has not written it to its logging
[ERROR] [ecj] The following line(s) might indicate the issue:
	byte[] bytes = IOUtils.toByteArray(jar.getInputStream(entry));
	               ^^^^^^^
IOUtils cannot be resolved
----------
35 problems (35 errors)

[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  19.201 s
[INFO] Finished at: 2024-07-04T22:12:27Z
[INFO] ------------------------------------------------------------------------
--snip--
```

This only covers 6 of the 29 `ecj-3.15.1` build failures, but there's little point examining the rest in detail right now:
```
wtwhite@wtwhite-vuw-vm:~/code/jcompile/runs/37_extend_36_with_matching_docker_images_for_ecj$ grep -l 'IOUtils cannot be resolved' `cat latest_errors.txt`|wc -l
6
wtwhite@wtwhite-vuw-vm:~/code/jcompile/runs/37_extend_36_with_matching_docker_images_for_ecj$ grep -L 'IOUtils cannot be resolved' `cat latest_errors.txt`|wc -l
23
```

The end result is that **`EQ.tsv` has not changed at all:**
```
wtwhite@wtwhite-vuw-vm:~/code/jcompile/runs/37_extend_36_with_matching_docker_images_for_ecj$ wc -l EQ.tsv 
465859 EQ.tsv
wtwhite@wtwhite-vuw-vm:~/code/jcompile/runs/37_extend_36_with_matching_docker_images_for_ecj$ wc -l IMPORTANT_LOGS_FROM_FIRST_OVERNIGHT_ECJ_RUN/EQ_JUSTINCASE.tsv 
465859 IMPORTANT_LOGS_FROM_FIRST_OVERNIGHT_ECJ_RUN/EQ_JUSTINCASE.tsv
wtwhite@wtwhite-vuw-vm:~/code/jcompile/runs/37_extend_36_with_matching_docker_images_for_ecj$ md5sum EQ.tsv !$
md5sum EQ.tsv IMPORTANT_LOGS_FROM_FIRST_OVERNIGHT_ECJ_RUN/EQ_JUSTINCASE.tsv
b4ab2e95b7ccf29d7ca0a2c97158f608  EQ.tsv
b4ab2e95b7ccf29d7ca0a2c97158f608  IMPORTANT_LOGS_FROM_FIRST_OVERNIGHT_ECJ_RUN/EQ_JUSTINCASE.tsv
```

On the bright side, this means there is no need to regenerate `NEQ1.tsv` or `NEQ2.tsv` 🤪

--- a/docker-build-project.sh
+++ b/docker-build-project.sh
@@ -96,7 +96,7 @@ mkdir -p $(dirname "${WORKTREE_HOST}")
 
 if test -n "$PREP_WORKTREE_CMD"; then
 	#( umask 0; cd "$WORKTREE_HOST"; $PREP_WORKTREE_CMD )
-	( umask 0; $PREP_WORKTREE_CMD "$WORKTREE_HOST" )
+	( umask 0; cd "$JCOMPILE_ROOT"; $PREP_WORKTREE_CMD "$WORKTREE_HOST" )
 fi
 
 MAVEN_HOST="$JCOMPILE_ROOT/apache-maven-3.9.2"

--- a/java-compilers.json
+++ b/java-compilers.json
@@ -81,20 +81,20 @@
 		"extra_mvn_args":"-Dmaven.compiler.target=20"
 	},
 	{
-		"image":"eclipse-temurin:8u372-b07-jdk",
-		"name":"ecj-3.11.1.v20150902-1521_openjdk-8.0.372",
+		"image":"eclipse-temurin:11.0.19_7-jdk",
+		"name":"ecj-3.11.1.v20150902-1521_openjdk-11.0.19",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.8.1",
 		"extra_mvn_args":"-Dmaven.compiler.target=1.8 -Dmaven.compiler.source=1.8"
 	},
 	{
-		"image":"eclipse-temurin:8u372-b07-jdk",
-		"name":"ecj-3.12.3_openjdk-8.0.372",
+		"image":"eclipse-temurin:11.0.19_7-jdk",
+		"name":"ecj-3.12.3_openjdk-11.0.19",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.8.2",
 		"extra_mvn_args":"-Dmaven.compiler.target=1.8 -Dmaven.compiler.source=1.8"
 	},
 	{
-		"image":"openjdk:9.0.1-jdk",
-		"name":"ecj-3.15.1_openjdk-9.0.1",
+		"image":"eclipse-temurin:11.0.19_7-jdk",
+		"name":"ecj-3.15.1_openjdk-11.0.19",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.8.6",
 		"extra_mvn_args":"-Dmaven.compiler.target=1.9 -Dmaven.compiler.source=1.9"
 	},

--- a/java-compilers.json
+++ b/java-compilers.json
@@ -84,49 +84,49 @@
 		"image":"eclipse-temurin:11.0.19_7-jdk",
 		"name":"ecj-3.11.1.v20150902-1521_openjdk-11.0.19",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.8.1",
-		"extra_mvn_args":"-Dmaven.compiler.target=1.8"
+		"extra_mvn_args":"-Dmaven.compiler.target=1.8 -Dmaven.compiler.source=1.8"
 	},
 	{
 		"image":"eclipse-temurin:11.0.19_7-jdk",
 		"name":"ecj-3.12.3_openjdk-11.0.19",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.8.2",
-		"extra_mvn_args":"-Dmaven.compiler.target=1.8"
+		"extra_mvn_args":"-Dmaven.compiler.target=1.8 -Dmaven.compiler.source=1.8"
 	},
 	{
 		"image":"eclipse-temurin:11.0.19_7-jdk",
 		"name":"ecj-3.15.1_openjdk-11.0.19",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.8.6",
-		"extra_mvn_args":"-Dmaven.compiler.target=1.9"
+		"extra_mvn_args":"-Dmaven.compiler.target=1.9 -Dmaven.compiler.source=1.9"
 	},
 	{
 		"image":"eclipse-temurin:11.0.19_7-jdk",
 		"name":"ecj-3.24.0_openjdk-11.0.19",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.9.0",
-		"extra_mvn_args":"-Dmaven.compiler.target=15"
+		"extra_mvn_args":"-Dmaven.compiler.target=15 -Dmaven.compiler.source=15"
 	},
 	{
 		"image":"eclipse-temurin:11.0.19_7-jdk",
 		"name":"ecj-3.28.0_openjdk-11.0.19",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.11.1",
-		"extra_mvn_args":"-Dmaven.compiler.target=17"
+		"extra_mvn_args":"-Dmaven.compiler.target=17 -Dmaven.compiler.source=17"
 	},
 	{
 		"image":"eclipse-temurin:11.0.19_7-jdk",
 		"name":"ecj-3.29.0_openjdk-11.0.19",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.12.0",
-		"extra_mvn_args":"-Dmaven.compiler.target=17"
+		"extra_mvn_args":"-Dmaven.compiler.target=17 -Dmaven.compiler.source=17"
 	},
 	{
 		"image":"eclipse-temurin:11.0.19_7-jdk",
 		"name":"ecj-3.30.0_openjdk-11.0.19",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.12.1",
-		"extra_mvn_args":"-Dmaven.compiler.target=18"
+		"extra_mvn_args":"-Dmaven.compiler.target=18 -Dmaven.compiler.source=18"
 	},
 	{
 		"image":"eclipse-temurin:11.0.19_7-jdk",
 		"name":"ecj-3.32.0_openjdk-11.0.19",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.13.0",
-		"extra_mvn_args":"-Dmaven.compiler.target=19"
+		"extra_mvn_args":"-Dmaven.compiler.target=19 -Dmaven.compiler.source=19"
 	},
 	{
 		"image":"container-registry.oracle.com/java/jdk:8u371-ol8-amd64",

--- a/java-compilers.json
+++ b/java-compilers.json
@@ -81,50 +81,50 @@
 		"extra_mvn_args":"-Dmaven.compiler.target=20"
 	},
 	{
-		"image":"eclipse-temurin:11.0.19_7-jdk",
-		"name":"ecj-3.11.1.v20150902-1521_openjdk-11.0.19",
+		"image":"eclipse-temurin:8u372-b07-jdk",
+		"name":"ecj-3.11.1.v20150902-1521_openjdk-8.0.372",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.8.1",
 		"extra_mvn_args":"-Dmaven.compiler.target=1.8 -Dmaven.compiler.source=1.8"
 	},
 	{
-		"image":"eclipse-temurin:11.0.19_7-jdk",
-		"name":"ecj-3.12.3_openjdk-11.0.19",
+		"image":"eclipse-temurin:8u372-b07-jdk",
+		"name":"ecj-3.12.3_openjdk-8.0.372",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.8.2",
 		"extra_mvn_args":"-Dmaven.compiler.target=1.8 -Dmaven.compiler.source=1.8"
 	},
 	{
-		"image":"eclipse-temurin:11.0.19_7-jdk",
-		"name":"ecj-3.15.1_openjdk-11.0.19",
+		"image":"openjdk:9.0.1-jdk",
+		"name":"ecj-3.15.1_openjdk-9.0.1",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.8.6",
 		"extra_mvn_args":"-Dmaven.compiler.target=1.9 -Dmaven.compiler.source=1.9"
 	},
 	{
-		"image":"eclipse-temurin:11.0.19_7-jdk",
-		"name":"ecj-3.24.0_openjdk-11.0.19",
+		"image":"openjdk:15.0.1-jdk",
+		"name":"ecj-3.24.0_openjdk-15.0.1",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.9.0",
 		"extra_mvn_args":"-Dmaven.compiler.target=15 -Dmaven.compiler.source=15"
 	},
 	{
-		"image":"eclipse-temurin:11.0.19_7-jdk",
-		"name":"ecj-3.28.0_openjdk-11.0.19",
+		"image":"eclipse-temurin:17.0.7_7-jdk",
+		"name":"ecj-3.28.0_openjdk-17.0.7",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.11.1",
 		"extra_mvn_args":"-Dmaven.compiler.target=17 -Dmaven.compiler.source=17"
 	},
 	{
-		"image":"eclipse-temurin:11.0.19_7-jdk",
-		"name":"ecj-3.29.0_openjdk-11.0.19",
+		"image":"eclipse-temurin:17.0.7_7-jdk",
+		"name":"ecj-3.29.0_openjdk-17.0.7",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.12.0",
 		"extra_mvn_args":"-Dmaven.compiler.target=17 -Dmaven.compiler.source=17"
 	},
 	{
-		"image":"eclipse-temurin:11.0.19_7-jdk",
-		"name":"ecj-3.30.0_openjdk-11.0.19",
+		"image":"eclipse-temurin:18.0.2_9-jdk",
+		"name":"ecj-3.30.0_openjdk-18.0.2",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.12.1",
 		"extra_mvn_args":"-Dmaven.compiler.target=18 -Dmaven.compiler.source=18"
 	},
 	{
-		"image":"eclipse-temurin:11.0.19_7-jdk",
-		"name":"ecj-3.32.0_openjdk-11.0.19",
+		"image":"eclipse-temurin:19.0.2_7-jdk",
+		"name":"ecj-3.32.0_openjdk-19.0.2",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.13.0",
 		"extra_mvn_args":"-Dmaven.compiler.target=19 -Dmaven.compiler.source=19"
 	},

--- a/java-compilers.json
+++ b/java-compilers.json
@@ -84,13 +84,13 @@
 		"image":"eclipse-temurin:11.0.19_7-jdk",
 		"name":"ecj-3.11.1.v20150902-1521_openjdk-11.0.19",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.8.1",
-		"extra_mvn_args":"-Dmaven.compiler.target=1.8 -Dmaven.compiler.source=1.8"
+		"extra_mvn_args":"-Dmaven.compiler.target=1.8"
 	},
 	{
 		"image":"eclipse-temurin:11.0.19_7-jdk",
 		"name":"ecj-3.12.3_openjdk-11.0.19",
 		"prep_worktree_cmd":"./inject-ecj-compiler.pl 2.8.2",
-		"extra_mvn_args":"-Dmaven.compiler.target=1.8 -Dmaven.compiler.source=1.8"
+		"extra_mvn_args":"-Dmaven.compiler.target=1.8"
 	},
 	{
 		"image":"eclipse-temurin:11.0.19_7-jdk",

